pkg/imagesystem/registrytemplate.go

```diff
@@ -102,7 +102,6 @@ func registryDeployment(namespace, registryImage string, requirements corev1.Res
 								FailureThreshold:    3,
 							},
 							SecurityContext: &corev1.SecurityContext{
-								RunAsUser:                z.Pointer[int64](1000),
```

Comment:
This probably breaks non-ocp k8s deploys. Ill probably have to create nonroot rbac for this and controller deployment.

Reply:
Made this change.